ArduCopter/Copter.cpp

```diff
@@ -166,7 +166,7 @@ const AP_Scheduler::Task Copter::scheduler_tasks[] = {
 #if RPM_ENABLED == ENABLED
     SCHED_TASK(rpm_update,            40,    200),
 #endif
-    SCHED_TASK(compass_cal_update,   100,    100),
+    SCHED_TASK_CLASS(Compass, &copter.compass, cal_update, 100, 100),
```

Comment:
Can this be moved to AP_Vehicle now?

Reply:
remember that if you move it you need to check whether its priority is changed and whether that affects anything.